### PR TITLE
br: support pause/resume stream task

### PR DESCRIPTION
**Pull request by joccau**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/33873

Problem Summary: As titile.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: br/pkg/stream/client.go [hunk 1]
```diff
@@ -106,6 +106,38 @@ func (c *MetaDataClient) GetTask(ctx context.Context, taskName string) (*Task, e
...
+	resps, err := c.KV.Txn(ctx).
+		Then(
+			clientv3.OpGet(TaskOf(taskName)),
+			clientv3.OpGet(Pause(taskName)),
+		).Commit()
+	if err != nil {
+		return nil, false, errors.Annotatef(err, "failed to fetch task %s", taskName)
+	}
+
+	if len(resps.Responses) == 0 {
```
Comment (3pointer) [inline br/pkg/stream/client.go:119]: if len(resps.Responses) == 0 || len(resps.Responses[0].GetResponseRange().Kvs) == 0 {

}

File: br/pkg/task/stream.go [hunk 2]
```diff
@@ -206,9 +227,25 @@ func (cfg *StreamConfig) ParseStreamStartFromFlags(flags *pflag.FlagSet) error {
 	}
 
 	if cfg.SafePointTTL <= 0 {
-		cfg.SafePointTTL = utils.DefaultStreamGCSafePointTTL
+		cfg.SafePointTTL = utils.DefaultBRGCSafePointTTL
```
Comment (3pointer) [inline br/pkg/task/stream.go:230]: should be DefaultStreamGCSafePointTTL?

**Review by 3pointer — APPROVED** [review]
LGTM

**Review by YuJuncen — APPROVED** [review]

**Pull request merged**